src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -35,27 +35,27 @@ interface IProps {
 }
 
 interface IThemeState {
-    theme: string,
-    useSystemTheme: boolean,
+    theme: string;
+    useSystemTheme: boolean;
 }
 
 export interface CustomThemeMessage {
-    isError: boolean,
-    text: string
+    isError: boolean;
+    text: string;
 };
 
 interface IState extends IThemeState {
     // String displaying the current selected fontSize.
     // Needs to be string for things like '17.' without
     // trailing 0s.
-    fontSize: string,
-    customThemeUrl: string,
-    customThemeMessage: CustomThemeMessage,
-    useCustomFontSize: boolean,
-    useIRCLayout: boolean,
+    fontSize: string;
+    customThemeUrl: string;
+    customThemeMessage: CustomThemeMessage;
+    useCustomFontSize: boolean;
+    useIRCLayout: boolean;
 }
 
-const MESSAGE_PREVIEW_TEXT = "Hey you. You're the best"
+const MESSAGE_PREVIEW_TEXT = _t("Hey you. You're the best!");
```

Comment:
this won't work as it'll be called before the language json files are loaded

Reply:
Can I do it as a class member? Or instead _td here and _t everywhere else?

Reply:
Depends on when the class is constructed, alternately the latter works too :)